Data/pubtator_to_xml.py

```diff
@@ -0,0 +1,152 @@
+import gzip
+
+import tqdm
+
+
+def generate_annotation(file_object, tagged_term, index):
```

Comment:
Would love if we could use an existing xml library so you don't have to hardcode the XML creation. Similar to using the `csv` module to write TSVs.